internal/cli/atlas_alerts_config_fields.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func AtlasAlertConfigsFieldsBuilder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "fields",
+		Aliases: []string{"field"},
+		Short:   "Manage alerts configuration fields for your project.",
```

Comment:
[nit] small grammatical nit, in english you only pluralize the last word
```suggestion
		Short:   "Manage alert configuration fields for your project.",
```